### PR TITLE
#1044: eclipse commandlet cleanup

### PR DESCRIPTION
**Pull request by Amueller36**
#1044 
Switched all variables to snake_case
Created Functions for the invocation of eclipse with specified arguments and extracting debug info.
Wrote functions for evaluating the debug info and log accordingly

### CODE REVIEW

File: scripts/src/main/resources/scripts/command/eclipse [hunk 1]
```diff
@@ -124,72 +126,104 @@ function doCheckPluginsInstalled() {
...
-  # Overwrite file with lines containing !MESSAGE
-  sed -i -n '/!MESSAGE/ p' "${debug_info_file}"
+  if [ $exit_code -ne 0 ];
+  then
+    doError "Failed to invoke Eclipse with arguments: ${eclipse_args[*]}"
+    echo ""
+    return 1
+  else
+  local debug_info
+  debug_info=$(<"${debug_info_file}")
```
Comment (hohwille) [inline scripts/src/main/resources/scripts/command/eclipse:196]: why do we first have to write the output into a file and then read the file into a variable and echo that?
IMHO this does not make sense anymore. Why don't we just remove `> "${debug_info_file}"` etc. and let the output go its regular way?

**Review by hohwille — COMMENTED** [review]
@Amueller36 thanks for your PR. Nice rework and as it seems you also fixed at least one bug 👍 
Can you check my comment and see if I am not mistaken?

File: scripts/src/main/resources/scripts/command/eclipse (hunk 1 above)
Comment (Amueller36) [reply scripts/src/main/resources/scripts/command/eclipse]: Done

**Review by hohwille — APPROVED** [review]
@Amueller36 thanks for this PR with the improvements. Looks good. I will merge via commandline and also do some testing when I have some time.

**Comment by hohwille** [comment]
Tested successfully and will now merge.

**Pull request merged**